corehq/apps/accounting/migrations/0066_remove_display_condition_on_nodeset_toggle.py

```diff
@@ -0,0 +1,33 @@
+from django.db import migrations
+
+from corehq.util.django_migrations import skip_on_fresh_install
+from corehq.toggles.models import Toggle
+from couchdbkit import ResourceNotFound
+
+TOGGLE_SLUG = "display_condition_on_nodeset"
+
+
+@skip_on_fresh_install
+def _remove_toggle_document(apps, schema_editor):
+    try:
+        toggle = Toggle.get(TOGGLE_SLUG)
+        if toggle:
```

Comment:
Maybe this check isn't necessary, since it will throw a `ResourceNotFound` error if it doesn't exist.